__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const request = require('supertest')
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+
+beforeEach(() => {
+    return seed(testData)
+    });
+
+  afterAll(() => {
+    db.end()
+  });
+
+describe("GET API/topics", () => {
+    test("Should return an array", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toBeInstanceOf(Array)
+        })
+    })
+    test("Array should have objects inside of it.", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body[0]).toBeInstanceOf(Object)
+        })
+    })
+    test("Objects in the array should have the predefined keys", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+           body.map((treasure) => {
+            expect(treasure).toHaveProperty("slug")
+            expect(treasure).toHaveProperty("description")
+           })
+        })
+    })
+    test("Objects in the array should have two properties", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+           body.map((treasure) => {
+            expect(Object.keys(treasure).length).toBe(2)
```

Comment:
It would be better to not restrict any data to only be a certain length, as it doesn't allow for future expansion/changes to the format of the data. What if in the future we decided there should be a third key added to the data in the database? This test would then fail, which means we would have to go back and edit an old test which is something we always want to avoid.